client/packages/invoices/src/Prescriptions/LineEditView/columns.tsx

```diff
@@ -150,9 +150,10 @@ export const usePrescriptionLineEditColumns = ({
 const UnitQuantityCell = (props: CellProps<DraftStockOutLineFragment>) => (
   <NumberInputCell
     {...props}
-    max={props.rowData.availablePacks}
+    max={props.rowData.availablePacks * props.rowData.packSize}
     decimalLimit={2}
     min={0}
+    debounce={1000}
     slotProps={{ htmlInput: { sx: { backgroundColor: 'white' } } }}
   />
 );
```

Comment:
Findings: 

`UnitQuantityCell` is part of columns which uses `draftLines`. Draft lines is updated through the `manualAllocate` method or the setter that is called on change for this column. But because of this the table will always re-render on update. Losing focus after typing/debounce times out. 

Attempted solutions like memoizing, followed patterns from `AutoAllocatePrescribedQuantity`, and additional debouncing don't work because draftLines fundamentally changes on every allocation update, forcing a re-render.

`AutoAllocatePrescribedQuantityField` doesn't experience this issue because prescribedQuantity is an independent state value that isn't affected by external allocation changes

We could use `useRef`, but feels like smelly code doing so

Reply:
Solution: Just make it debounce for 1000, which seems to be enough time to type

Reply:
The actual issue of losing focus is coming from DataRow. Getting fixed in a separate issue